src/Shared/Model/PackageMetadata.cs

```diff
@@ -177,6 +178,17 @@ public class User
 
         [JsonProperty(PropertyName = "url", NullValueHandling = NullValueHandling.Ignore)]
         public string? Url { get; set; }
+
+        protected bool Equals(User other) => Active == other.Active && Email == other.Email && Id == other.Id && Name == other.Name && Url == other.Url;
+
+        public override bool Equals(object? obj)
+        {
+            if (ReferenceEquals(null, obj)) return false;
+            if (ReferenceEquals(this, obj)) return true;
+            return obj.GetType() == this.GetType() && Equals((User) obj);
+        }
+
+        public override int GetHashCode() => HashCode.Combine(Active, Email, Id, Name, Url);
```

Comment:
Nit: Any chance that this class could be made an [immutable record type](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/record) instead? We might be able to get these definitions for free.

Reply:
Resolved in other PR.